hed/schema/schema_attribute_validators.py

```diff
@@ -165,16 +137,7 @@ def unit_exists(hed_schema, tag_entry, attribute_name):
 
 # This is effectively unused and can never fail - The schema would catch these errors and refuse to load
 def tag_exists_base_schema_check(hed_schema, tag_entry, attribute_name):
-    """ Check if the single tag is a partnered schema tag
-
-    Parameters:
-        hed_schema (HedSchema): The schema to use for validation
-        tag_entry (HedSchemaEntry): The schema entry for this tag.
-        attribute_name (str): The name of this attribute
-
-    Returns:
-        list: A list of issues. Each issue is a dictionary.
-    """
+    """Check if the single tag is a partnered schema tag"""
```

Comment:
Why is the documentation of the Parameters and Returns being removed?

Reply:
Because it was the exact same on all of those functions, there's a template at the top of the file(I just fixed a mistake in it).  Users would never be calling these functions directly.

I'm happy to add the parameter/return value doc strings back if desired, they just didn't seem helpful to me.

Reply:
Please put them back in.  When you jump to a particular function from a link in the docs you don't see the template.

Reply:
Done